detections/endpoint/recon_avproduct_through_pwh_or_wmi.yml

```diff
@@ -15,73 +14,71 @@ description:
   aiding in evasion techniques. If confirmed malicious, this could allow attackers
   to disable or bypass security measures, leading to further compromise of the endpoint.
 data_source:
-  - Powershell Script Block Logging 4104
-search:
-  '`powershell` EventCode=4104 (ScriptBlockText = "*SELECT*" OR ScriptBlockText
+- Powershell Script Block Logging 4104
```

Comment:
A lot of changes are also related to indentation. Which we want to have. So if you can avoid changing that